apps/gitness/src/routes.tsx

```diff
@@ -389,33 +389,38 @@ export const routes: CustomRouteObject[] = [
                   },
                   {
                     path: 'commits',
-                    element: <RepoCommitsPage />,
                     handle: {
                       breadcrumb: () => <Text>Commits</Text>,
                       routeName: RouteConstants.toRepoCommits
-                    }
-                  },
-                  {
-                    path: 'commits/:commitSHA',
-                    element: <RepoCommitDetailsPage />,
+                    },
                     children: [
                       {
                         index: true,
-                        element: <RepoCommitDiffsPage />,
+                        element: <RepoCommitsPage />
+                      },
+                      {
+                        path: ':commitSHA',
+                        element: <RepoCommitDetailsPage />,
                         handle: {
                           breadcrumb: ({ commitSHA }: { commitSHA: string }) => (
                             <>
-                              <Text>Commits</Text>
-                              <Breadcrumb.Separator />
-                              <Text>{commitSHA}</Text>
+                              <Text>{commitSHA.substring(0, 7)}</Text>
```

Comment:
```suggestion
                              <span>{commitSHA.substring(0, 7)}</span>
```
Please remember that we should avoid using the `Text` component, as it’s planned to be removed in the future